edgexfoundry/edgex-compose
# feat: Pin 3rd party Docker images for Napa

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions


1. run `make run no-secty` to start services in non-secure mode and ensure that the 3rd party images all have a pinned version
2. repeat the same with `make run` in secure mode
